compiler/testData/diagnostics/tests/resolve/CycleInTypeArgs.kt

```diff
@@ -1,3 +1,4 @@
+// FIR_IDENTICAL
```

Comment:
Please squash this `FIR_IDENTICAL` commit with other commits.

Reply:
Yeah, I saw that as well: https://github.com/JetBrains/kotlin/commit/f2c1608c2bb6f33ed41dc79b14f7b295cc8ca133

Alas, right after that, one more is coming :\

https://github.com/JetBrains/kotlin/commit/f06a5321a62585741c99a69c0a7b84d6e03b0927#diff-c71133511366bfd95dc5b8ac8c94b606209c7f327220c5b6b5972be3c6a7e984